lib/dpl/provider/elastic_beanstalk.rb

```diff
@@ -103,7 +103,7 @@ def bucket_exists?
       end
 
       def create_bucket
-        s3.buckets.create(bucket_name)
+        s3.create_bucket({ bucket: bucket_name })
```

Comment:
Haha, I agree. That's what happens when you spend years writing Javascript and haven't done Ruby in a long time. I will update the PR.